2022-05-05 22:15:37,525 INFO    MainThread:5217 [wandb_setup.py:_flush():75] Loading settings from /Users/zhangjessie/.config/wandb/settings
2022-05-05 22:15:37,525 INFO    MainThread:5217 [wandb_setup.py:_flush():75] Loading settings from /Users/zhangjessie/PycharmProjects/py3.7/spinup/algos/pytorch/sac/wandb/settings
2022-05-05 22:15:37,525 INFO    MainThread:5217 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-05-05 22:15:37,525 INFO    MainThread:5217 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'sac.py', 'program': '/Users/zhangjessie/PycharmProjects/py3.7/spinup/algos/pytorch/sac/sac.py'}
2022-05-05 22:15:37,525 INFO    MainThread:5217 [wandb_init.py:_log_setup():437] Logging user logs to /Users/zhangjessie/PycharmProjects/py3.7/spinup/algos/pytorch/sac/wandb/run-20220505_221537-2uwc3jiq/logs/debug.log
2022-05-05 22:15:37,526 INFO    MainThread:5217 [wandb_init.py:_log_setup():438] Logging internal logs to /Users/zhangjessie/PycharmProjects/py3.7/spinup/algos/pytorch/sac/wandb/run-20220505_221537-2uwc3jiq/logs/debug-internal.log
2022-05-05 22:15:37,526 INFO    MainThread:5217 [wandb_init.py:init():471] calling init triggers
2022-05-05 22:15:37,526 INFO    MainThread:5217 [wandb_init.py:init():475] wandb.init called with sweep_config: {}
config: {}
2022-05-05 22:15:37,526 INFO    MainThread:5217 [wandb_init.py:init():524] starting backend
2022-05-05 22:15:37,526 INFO    MainThread:5217 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-05-05 22:15:37,534 INFO    MainThread:5217 [backend.py:ensure_launched():217] starting backend process...
2022-05-05 22:15:37,541 INFO    MainThread:5217 [backend.py:ensure_launched():222] started backend process with pid: 5224
2022-05-05 22:15:37,544 INFO    MainThread:5217 [wandb_init.py:init():533] backend started and connected
2022-05-05 22:15:37,553 INFO    MainThread:5217 [wandb_init.py:init():597] updated telemetry
2022-05-05 22:15:37,555 INFO    MainThread:5217 [wandb_init.py:init():628] communicating run to backend with 30 second timeout
2022-05-05 22:15:38,811 INFO    MainThread:5217 [wandb_run.py:_on_init():1923] communicating current version
2022-05-05 22:15:40,859 INFO    MainThread:5217 [wandb_run.py:_on_init():1927] got version response upgrade_message: "wandb version 0.12.16 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-05-05 22:15:40,860 INFO    MainThread:5217 [wandb_init.py:init():659] starting run threads in backend
2022-05-05 22:15:42,241 INFO    MainThread:5217 [wandb_run.py:_console_start():1897] atexit reg
2022-05-05 22:15:42,242 INFO    MainThread:5217 [wandb_run.py:_redirect():1770] redirect: SettingsConsole.REDIRECT
2022-05-05 22:15:42,242 INFO    MainThread:5217 [wandb_run.py:_redirect():1775] Redirecting console.
2022-05-05 22:15:42,244 INFO    MainThread:5217 [wandb_run.py:_redirect():1831] Redirects installed.
2022-05-05 22:15:42,244 INFO    MainThread:5217 [wandb_init.py:init():684] run started, returning control to user process
2022-05-05 22:19:04,449 INFO    MainThread:5217 [wandb_setup.py:_flush():75] Loading settings from /Users/zhangjessie/.config/wandb/settings
2022-05-05 22:19:04,449 INFO    MainThread:5217 [wandb_setup.py:_flush():75] Loading settings from /Users/zhangjessie/PycharmProjects/py3.7/spinup/algos/pytorch/sac/wandb/settings
2022-05-05 22:19:04,449 INFO    MainThread:5217 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-05-05 22:19:04,450 INFO    MainThread:5217 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'sac.py', 'program': '/Users/zhangjessie/PycharmProjects/py3.7/spinup/algos/pytorch/sac/sac.py'}
2022-05-05 22:19:04,450 INFO    MainThread:5217 [wandb_init.py:_log_setup():437] Logging user logs to /Users/zhangjessie/PycharmProjects/py3.7/spinup/algos/pytorch/sac/wandb/run-20220505_221904-1tig37em/logs/debug.log
2022-05-05 22:19:04,450 INFO    MainThread:5217 [wandb_init.py:_log_setup():438] Logging internal logs to /Users/zhangjessie/PycharmProjects/py3.7/spinup/algos/pytorch/sac/wandb/run-20220505_221904-1tig37em/logs/debug-internal.log
2022-05-05 22:19:04,450 INFO    MainThread:5217 [wandb_init.py:init():471] calling init triggers
2022-05-05 22:19:04,450 INFO    MainThread:5217 [wandb_init.py:init():475] wandb.init called with sweep_config: {}
config: {}
2022-05-05 22:19:04,450 INFO    MainThread:5217 [wandb_init.py:init():519] wandb.init() called when a run is still active
2022-05-05 22:20:33,483 INFO    MainThread:5217 [wandb_run.py:_atexit_cleanup():1866] got exitcode: 255
2022-05-05 22:20:33,484 INFO    MainThread:5217 [wandb_run.py:_restore():1838] restore
2022-05-05 22:20:35,777 INFO    MainThread:5217 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 531
  total_bytes: 531
}

2022-05-05 22:20:35,949 INFO    MainThread:5217 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 531
  total_bytes: 531
}

2022-05-05 22:20:36,657 INFO    MainThread:5217 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 531
  total_bytes: 14305
}

2022-05-05 22:20:36,762 INFO    MainThread:5217 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 531
  total_bytes: 14420
}

2022-05-05 22:20:36,868 INFO    MainThread:5217 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 531
  total_bytes: 14420
}

2022-05-05 22:20:36,975 INFO    MainThread:5217 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2605
  total_bytes: 14420
}

2022-05-05 22:20:37,079 INFO    MainThread:5217 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2605
  total_bytes: 14420
}

2022-05-05 22:20:37,185 INFO    MainThread:5217 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2605
  total_bytes: 14420
}

2022-05-05 22:20:37,291 INFO    MainThread:5217 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2605
  total_bytes: 14420
}

2022-05-05 22:20:37,397 INFO    MainThread:5217 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 14420
  total_bytes: 14420
}

